kubernetes/samples/scripts/create-weblogic-domain/model-in-image/README.md

```diff
@@ -0,0 +1,9 @@
+# Model in Image Sample
+
+This sample demonstrates the WebLogic Kubernetes Operator "Model in Image" feature. Model in Image allows you to supply a Weblogic Deploy Tool (WDT) model that the operator automatically expands into a full domain home during runtime. This eliminates the need to pre-create a WebLogic domain home prior to deploying your domain resource.
+
+# References
+
+- Sample doc: [source](../../../../../docs-source/content/samples/simple/domains/model-in-image/_index.md), [latest public](https://oracle.github.io/weblogic-kubernetes-operator/samples/simple/domains/model-in-image).
```

Comment:
I think there is a paradigm for relative links I saw earlier... Might be less cumbersome than this.